{
    "PackageVulnerabilityID": "1",
    "DisclosureDate": "2021-01-28",
    "AffectedVersionRange": "*",
    "Summary": "Arbitrary Code Execution",
    "Contributor": {
        "Discloser": "61599526",
        "Fixer": ""
    },
    "Package": {
        "Registry": "pip",
        "Name": "tfx",
        "URL": "https://pypi.org/project/tfx",
        "Downloads": "11193"
    },
    "CWEs": [
        {
            "ID": "94",
            "Description": "Arbitrary Code Execution"
        }
    ],
    "CVSS": {
        "Version": "3.1",
        "AV": "P",
        "AC": "H",
        "PR": "L",
        "UI": "N",
        "S": "U",
        "C": "L",
        "I": "L",
        "A": "L",
        "E": "",
        "RL": "",
        "RC": "",
        "Score": "3.8"
    },
    "CVEs": [
        ""
    ],
    "Repository": {
        "URL": "https://github.com/tensorflow/tfx",
        "Codebase": [
            "Python"
        ],
        "Owner": "tensorflow",
        "Name": "tfx",
        "Forks": "396",
        "Stars": "1319"
    },
    "Permalinks": [
        "https://drive.google.com/file/d/1M8OVLlrnrmujCWmq5zmRGxXRAyJCSWxq/view?usp=sharing"
    ],
    "References": [
        {
            "Description": "",
            "URL": ""
        }
    ],
    "PrNumber": "1803",
    "FixSubmissionCount": 1
}